src/compiler/sys/logger/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../../../tsconfig-base.json",
+  "compilerOptions": {
+    "composite": true,
+    "strict": true
+  },
+  "include": [
+    "./index.ts",
+    "./logger.ts",
+    "./console-logger.ts",
+    "./terminal-logger.ts"
+  ],
+  "references": [
+    { "path": "../environment" },
+    { "path": "../../diagnostic" }
+  ]
```

Comment:
this project is a little more substantial. It includes four files and depends on (`'references'`) the `environment` and `diagnostic` projects.

Note also that we have `"strict": true` here, so the files under here are now somewhat permanently strictly-checked.